features/MiniFarms/JarCollapsible.tsx

```diff
@@ -17,6 +25,9 @@ import { JarApy } from "./MiniFarmList";
 import { useTranslation } from "next-i18next";
 import { isUsdcToken } from "containers/Jars/jars";
 import { PickleCore } from "containers/Jars/usePickleCore";
+import { TokenDetails } from "containers/Jars/useJarsWithZap";
+import { formatUnits } from "picklefinance-core/node_modules/@ethersproject/units";
```

Comment:
@UrAvgDeveloper this is the import that breaks the build. Please only import from top-level packages.